### DISCUSSION TITLE
can't resolve reference #/definitions/directiveConfigSchema from id #

### DISCUSSION THREAD

**Discussion by gaoyusongcn** [quasarframework/quasar#14245] [CLI - PWA mode] [answered]
Hi there

Please help me. How can I fix this prompt...

![image](https://user-images.githubusercontent.com/43459833/186231447-835de7ad-d6f3-4ffa-a5a9-40fde6d344ba.png)

**Comment by MilosPaunovic** [comment]
A quick Google search suggests that this is related to ESLint. Provide more details for better support.

**Comment by AndreMPCosta** [comment]
I'm having the same problem, with default quasar config.

**Reply by gaoyusongcn** [reply]
yes. I don't know what's happened

**Comment by MilosPaunovic** [comment, marked as answer]
This is an ESLint issue: https://github.com/typescript-eslint/typescript-eslint/issues/5525.
It's resolved in [v5.35.1](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v5.35.1), so you should just update that package using package manager of your choice.

**Reply by gaoyusongcn** [reply]
Thank you
